### ISSUE TITLE
MediaStreamTrack.getSources is deprecated in chrome

### ISSUE THREAD

**Issue by nasr18** [bahmutov/ng-simple-webrtc#18]
I'm getting above error while running on chrome and firefox is not supporting `MediaStreamTrack.getSources`. pls do something.

**Comment by samsont** [comment]
Can you do some searching for us? Let's take chrome: What version of chrome are you using? And, please look up what version is it supported in. Thanks.

**Comment by nasr18** [comment]
actually i fixed it. after changing video size (450px) other than 1280 it works.

**Comment by nasr18** [comment]
i can broadcast the video(no audio) but cant able to watch the room. getting `nick` undefined. anyway to fix this???

**Comment by samsont** [comment]
Look at this ticket and try the version that fixed his problem: https://github.com/bahmutov/ng-simple-webrtc/issues/15 . Maybe even earlier version. We haven't use this to test audio in a while.

**Comment by nasr18** [comment]
yeah i too after checkin that release now watch working for me. but no audio. pls fix that.

**Comment by amanuel2** [comment]
Sup Nasr @nasr18

**Comment by bahmutov** [comment]
Pull requests with fix are welcome!

**Comment by amanuel2** [comment]
https://github.com/bahmutov/ng-simple-webrtc/pull/19

**Comment by nasr18** [comment]
still not working. @amanuel2

**Comment by fusion2013** [comment]
hi I resolved above issue by changing following code in broadcast html file :

if (window.MediaStreamTrack) {
          MediaStreamTrack.getSources(function (sources) {
            var videoSources = sources.filter(function (source) {
              return source.kind === 'video';
            });
            console.log('got video sources', videoSources);
          });
          }

use below  

  if (navigator.mediaDevices) {
              navigator.mediaDevices.enumerateDevices()
                  .then(function (sources) {
                      var videoSources = sources.filter(function (source) {
                          return source.kind === 'video';
                      });
                      console.log('got video sources', videoSources);
                  })
          }
